File: Java/Card.java
import java.util.Date;

class Card extends Payment {

    Integer number;
    Integer cvv;
    Date date;
    public Card(Integer id, Integer number, Integer cvv, Date date){
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}

File: PHP/Card.php
<?php

require_once('Payment.php');
class Card extends Payment {
    public $number
    public $cvv
    public $date;

    public function __construct($id, $number, $cvv, $date){
        $this->number = $number
        $this->cvv = $cvv
        $this->date = $date;
    }
}
?>